plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/AbstractSQLDialect.java

```diff
@@ -743,6 +743,11 @@ protected String getStoredProcedureCallInitialClause(DBSProcedure proc) {
         }
     }
 
+    @NotNull
+    protected String getProcedureCallEndClause(DBSProcedure procedure) {
+        return "";
+    }
+
```

Comment:
Consider moving this method to the super interface [`SQLDialect`](https://github.com/dbeaver/dbeaver/blob/devel/plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLDialect.java)

Reply:
I'm not sure about this.
getStoredProcedureCallInitialClause not in SQLDialect.

Reply:
Okay, I see.